src/witan/sen2/return/person_level/blade_export/csv_eda.clj

```diff
@@ -0,0 +1,379 @@
+                         (tc/set-dataset-name "`sen-need` (with ancestor `table-id`s)"))]
+    (clerk/fragment
+     (report-unique-key? named-plan   [:person-table-id :requests-table-id])
+     (report-unique-key? named-plan   [:person-table-id])
+     (report-unique-key? active-plans [:person-table-id :requests-table-id])
+     (report-unique-key? active-plans [:person-table-id])
+     (report-unique-key? sen-need     [:person-table-id :requests-table-id :sen-type-rank])
+     (report-unique-key? sen-need     [:person-table-id :sen-type-rank]))))
+
+
+
+;;; # Relational database structure: schema and `table-id` key relationships
+(defn report-expected-schema
+  []
+  (clerk/fragment
+   (clerk/md "Drawn using [asciiflow](https://asciiflow.com/#/share/eJytUkFuwjAQ%2FEq0V7AUm6aHfYsvi7MSSIlLsy4CIW59QhX%2BwbHqa3hJ7VSibiFRD135sJ4Zr0djH8BTy4Ag7I1ysi3ErbglmENDe%2B4ic7Cws4C6LBdzC%2FvYmkcTu8C7EDcWiqxKTIOs9cWPuvTnW%2Bzt4xemccOdPPm%2FHtezLzS291f%2FbrDj5xeWIJfT65juOqUYrVxVKj2tWiCJsEjLPkxM6s%2F%2FcF8ko%2FCBML1jrTYN%2BZzOs6gIyYX1lgfVdyATdsYyye1pZW5ZPbt%2FKoqT3SUmC6rmQOtm4KoBcpwyu%2BKVS99JeebawhGOn8S42w8%3D%29):")
+   (clerk/md (str " ```                                                                          \n"
+                  "                            0:sen2                                            \n"
+                  "                              ^                                               \n"
+                  "                              │                                               \n"
+                  "                            1:person                                          \n"
+                  "                              ^                                               \n"
+                  "                              │1+                                             \n"
+                  "                   ┌───────>2:requests<───────┐                               \n"
+                  "                   │                          │                               \n"
+                  "                   │0-1                       │                               \n"
+                  "                 3:assessment                 │                               \n"
+                  "                   ^                          │                               \n"
+                  "                   │0-1                       │0-1                            \n"
+                  "                4a:named-plan         ┌───>5a:active-plans<─┐                 \n"
+                  "                   ^                  │                     │                 \n"
+                  "                   │1-2               │1+                   │1-2              \n"
+                  "                4b:plan-detail     5b:placement-detail   5c:sen-need          \n"
+                  " ```                                                                          \n"))
```

Comment:
this is good documentation, but we really should find a prettier way to do this at some point